packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -86,25 +96,19 @@ export default class HostItem extends Component {
                 &nbsp;&nbsp;
                 <Tooltip
                   content={
-                    isEmpty(host.current_operations) ? (
-                      toolTipContent
-                    ) : (
-                      <div>
-                        {toolTipContent}
-                        {' ('}
-                        {map(host.current_operations)[0]}
-                        {')'}
-                      </div>
-                    )
+                    <span>
+                      {_(`powerState${state}`)}
+                      {state === 'Busy' && (
+                        <span>
+                          {' ('}
+                          {map(host.current_operations)[0]}
```

Comment:
Did you want to create a component that takes VM/HOST objects and render there titles ?

Reply:
It could be a good idea, but we would need to handle the "pill vs icon" problem. Anyway: not in this PR that is already too long.

Reply:
I resolve it in favor of the [comment](https://github.com/vatesfr/xen-orchestra/pull/4363#discussion_r305396080).